src/app.ts

```diff
@@ -211,22 +212,22 @@ export const start = async (args) => {
 
       process.exit(0);
     } catch (error) {
-      if (error !== "SIGTERM") {
+      if ((error as string) !== "SIGTERM") {
         screen.destroy();
 
         process.stderr.write(
           chalk.bold.red("[ERR] ") +
             "Unable to checkout " +
             chalk.yellow(gitBranch) +
             "\n" +
-            error.toString(),
+            (error as string),
         );
 
         process.exit(1);
       } else {
         refreshTable();
 
-        logger.error(error);
+        logger.error(error as string);
```

Comment:
It feels like using `(error as Error).message` everywhere you've got `error as string` is a more correct Typescript way of doing this. I'd suggest tweaking to that (you've already done that in `state.ts` so would make things more consistent).

Reply:
@rahulk94 I fully agree. 

However, I did play around with changing the git api to reject with actual `Error` types, but decided against it as that is not the intent of this effort.

IMO we should add that change under a separate issue, but I'll defer to @jwu910 for the final verdict.

Reply:
It's been a long time, but if I'm remembering this part is really just for notifying events in the log, not entirely sure it will be absolutely necessary at least in this space to catch the `Error` type since it's just a message to the user.

But maybe we can change the variable name to errorMessage in the future to be more clear